tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -799,3 +799,9 @@ test_i2c_tsl2561: tsl2561@78 {
 	compatible = "ams,tsl2561";
 	reg = <0x78>;
 };
+
+test_i2c_bmp581: bmp581@79 {
+	compatible = "bosch,bmp581";
+	reg = <0x70>;
```

Comment:
should be 0x79

Reply:
fixed

Reply:
Rebased again and increased to 0x80 due to merge conflict.